serverFiles/models/articles.models.js

```diff
@@ -18,14 +18,43 @@ function selectArticlesById(article_id) {
     });
 }
 
-function selectAllArticles() {
+function selectAllArticles(filterBy, sortBy, orderBy) {
+  const validSortBy = ['comment_count', 'author', 'title', 'article_id', 'topic', 'created_at', 'votes', 'article_img_url']
+    if(!validSortBy.includes(sortBy)) {
+      return Promise.reject({status:400,msg:'Bad request'})
+    }
+
+  const validFilterBy = ['mitch', 'cats', 'paper']
```

Comment:
This fixed list might cause you some trouble when you host as the development data topics are different to the test data topics. It might be good to make a database call to check if your row's are empty because the topic is invalid or if the topic just doesn't have any articles yet.